numpy/lib/tests/test_function_base.py

```diff
@@ -2311,6 +2311,33 @@ def test_array_like(self):
         assert_array_equal(y1, y2)
         assert_array_equal(y1, y3)
 
+    def test_bool_dtypes(self, dtype):
```

Comment:
Oops sorry. Left the `dtype` in by accident. We can drop that

```suggestion
    def test_bool_dtypes(self):
```

This will fix the CI error